### ISSUE TITLE
add syntax highlighting for new %w verb

### ISSUE THREAD

**Issue by carnott-snap** [fatih/vim-go#2344]
### Background
go1.13 will support new error handling semantics, currently in preview via the [`xerrors`](https://godoc.org/golang.org/x/xerrors) package. One of these changes is the new `%w` verb for `fmt.Errorff`, currently exposed via [`xerrors.Errorf`](https://godoc.org/golang.org/x/xerrors#Errorf):
```go
func Errorf(format string, a ...interface{}) error
    Errorf formats according to a format specifier and returns the string as a
    value that satisfies error.

    The returned error includes the file and line number of the caller when
    formatted with additional detail enabled. If the last argument is an error
    the returned error's Format method will return it if the format string ends
    with ": %s", ": %v", or ": %w". If the last argument is an error and the
    format string ends with ": %w", the returned error implements Wrapper with
    an Unwrap method returning it.
```

### What did you do? (required: The issue will be **closed** when not provided)
Typed the following:
```
xerrors.Errorf("my error description: %w", err)
```

### What did you expect to happen?
See syntax highlighting on `%w` turn red like `%v`, `%s`, etc.

### What happened instead?
No syntax highlighting.

### Configuration (**MUST** fill this out):
```vimrc
filetype plugin indent on
```

#### vim-go version:
255f903c14350fb5d615b316661bac93db1544ce

####  `vimrc` you used to reproduce (use a *minimal* vimrc with other plugins disabled; do not link to a 2,000 line vimrc):
<details><summary>vimrc</summary><br><pre>
filetype plugin indent on
</pre></details>

#### Vim version (first three lines from `:version`):
8.1

#### Go version (`go version`):
go1.12.5

#### Go environment
<details><summary><code>go env</code> Output:</summary><br><pre>
GOARCH="amd64"
GOBIN=""
GOCACHE="/home/user/.cache/go-build"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOOS="linux"
GOPATH="/home/user/go"
GOPROXY=""
GORACE=""
GOROOT="/home/user/.local/share/umake/go/go-lang"
GOTMPDIR=""
GOTOOLDIR="/home/user/.local/share/umake/go/go-lang/pkg/tool/linux_amd64"
GCCGO="gccgo"
CC="gcc"
CXX="g++"
CGO_ENABLED="1"
GOMOD="/home/user/go.mod"
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build468235942=/tmp/go-build -gno-record-gcc-switches"
</pre></details>

**Comment by bhcleek** [comment]
Go tip is not officially supported by vim-go.

**Comment by bhcleek** [comment]
Go tip is not officially supported by vim-go.

**Comment by carnott-snap** [comment]
@bhcleek [1.13](https://golang.org/doc/go1.13) is out

**Comment by carnott-snap** [comment]
nevermind, looks like this got fixed in 68126af7941994658e16af912ae1b51b7e06a47d
